rootfs/bin/create_bucket

```diff
@@ -85,6 +85,6 @@ else:
         port=int(os.getenv('S3_PORT')),
         calling_format=OrdinaryCallingFormat())
     # HACK(bacongobbler): allow boto to connect to minio by changing the region name for s3v4 auth
-    conn.auth_region_name = os.getenv('AWS_REGION')
+    conn.auth_region_name = os.getenv('S3_REGION')
```

Comment:
I like the ENV var name change as it is better naming style to name it `S3_REGION` instead of `AWS_REGION` :+1:

Reply:
Looks good!